src/turbopuffer.test.ts

```diff
@@ -128,7 +128,7 @@ test("sanity", async () => {
   }
   expect(gotError).toStrictEqual(
     new TurbopufferError(
-      "💔 Resource not found; the requested namespace is empty.",
+      "🤷 namespace 'typescript_sdk_sanity' was not found",
```

Comment:
Unrelated to the other changes in this PR but this test was legitimately broken because we changed the response message server-side a while ago and never updated it here.